CHANGELOG.md

```diff
@@ -59,3 +59,5 @@ significant modifications will be credited to OpenTelemetry Authors.
 ([#260](https://github.com/open-telemetry/opentelemetry-demo/pull/260))
 * Added span attributes to currency service
 ([#265](https://github.com/open-telemetry/opentelemetry-demo/pull/265))
+* 39 Moving the Front end application from Go to Next.js
```

Comment:
This changelog entry doesn't make much sense to me, consider making the following changes:
1. Either remove `39` or make it an actual link to the issue.
2. Use passive voice (e.g. added, redesigned, reimplemented) rather than `Moving...`.
3. Focus on **what's the takeaway**, a developer reading this entry needs to get a good idea what this is.

Reply:
Hey @reyang thanks for the feedback, I just updated it, let me know what you think